frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx

```diff
@@ -382,31 +389,33 @@ function MobileView(props: CodingBountiesProps) {
         titleString={titleString}
         showPayBounty={showPayBounty}
         markUnpaid={
-          <IconButton
-            width={'100%'}
-            height={48}
-            style={{
-              bottom: '10px',
-              border: `1px solid ${color.primaryColor.P400}`,
-              background: color.pureWhite,
-              color: color.borderGreen1
-            }}
-            text={'Mark Unpaid'}
-            loading={saving === 'paid' || updatingPayment}
-            endingImg={'/static/mark_unpaid.svg'}
-            textStyle={{
-              width: '130px',
-              display: 'flex',
-              justifyContent: 'center',
-              fontFamily: 'Barlow',
-              marginLeft: '30px',
-              fontSize: '15px'
-            }}
-            onClick={handleSetAsUnpaid}
-          />
+          hasAccess && (
+            <IconButton
+              width={'100%'}
+              height={48}
+              style={{
+                bottom: '10px',
+                border: `1px solid ${color.primaryColor.P400}`,
+                background: paidStatus ? color.green1 : color.pureWhite,
+                color: paidStatus ? color.white100 : color.borderGreen1
+              }}
+              text={paidStatus ? 'Mark as Unpaid' : 'Mark as Paid'}
```

Comment:
lets make the text `Mark as Unpaid` and `Mark as Paid` a variable and then use them in the different places so if we decide to change the text it easier to change everywhere